assets/src/blocks/panel/edit.js

```diff
@@ -55,8 +56,57 @@ class StorytimePanel extends Component {
 		super( ...arguments )
 	}
 
+	getNextPanelId() {
+		// Get the id of the next panel in here doi
+		return '12378462834623';
+	}
+
 	render() {
 
+		/**
+		 * Need to create buttons obj based on:
+		 *
+		 * if next panel is a nested panel,
+		 * 	get the each nested panel and generate a button for each
+		 * else it is a single panel
+		 * 	generate a button for the single panel
+		 *
+		 **/
+
+		 const { attributes, setAttributes } = this.props;
+
+		// This code is essentially redundant to what is in PHP.
```

Comment:
Yeah. So there are now _three_ areas where we plug data into the pattern object – PHP, Node in the pattern prototype, and now in the edit method.